### PR TITLE
Clean up workshop tutorial

### PR DESCRIPTION
**Pull request by TimBartholomew**
## Fixes/Resolves:
None

## Summary/Motivation:
The tutorial folder had old files used to make the AEESP workshop and an incomplete tutorial. This PR removes the unnecessary files and updates the tutorial to the final version used at the workshop.

## Changes proposed in this PR:
- Adds an introduction.ipynb to tutorials which replaces the incomplete AEESP workshop tutorial
- Removes all the extraneous files from tutorials

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Comment by TimBartholomew** [comment]
@bknueven @dangunter I think this is good to go or at least is much better than what was there before.

One thing to keep in mind is that there are the deprecation warnings for passing keywords to the units, I would think that @lbianchi-lbl's automation to address that won't do it for Jupyter notebooks.

File: tutorials/introduction.ipynb [hunk 1]
```diff
@@ -0,0 +1,1892 @@
...
+      "\u001b[1;32mc:\\users\\tim\\.conda\\envs\\watertap-dev\\lib\\site-packages\\pyomo\\core\\base\\block.py\u001b[0m in \u001b[0;36m__getattr__\u001b[1;34m(self, val)\u001b[0m\n\u001b[0;32m    617\u001b[0m         \u001b[1;31m# Since the base classes don't support getattr, we can just\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    618\u001b[0m         \u001b[1;31m# throw the \"normal\" AttributeError\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m--> 619\u001b[1;33m         raise AttributeError(\"'%s' object has no attribute '%s'\"\n\u001b[0m\u001b[0;32m    620\u001b[0m                              % (self.__class__.__name__, val))\n\u001b[0;32m    621\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n",
+      "\u001b[1;31mAttributeError\u001b[0m: '_ScalarWaterTAPCosting' object has no attribute 'reverse_osmosis_membrane_cost'"
+     ]
+    }
+   ],
+   "source": [
+    "# double the membrane costs\n",
+    "m.fs.costing.reverse_osmosis_membrane_cost.fix(60)\n",
```
Comment (bknueven) [inline tutorials/introduction.ipynb]:
```suggestion
    "m.fs.costing.reverse_osmosis.membrane_cost.fix(60)\n",
```

**Review by bknueven — CHANGES_REQUESTED** [review]
The last cell of the introduction notebook has an attribute error. You can fix it by applying my suggestion and re-running all cells.

File: tutorials/introduction.ipynb (hunk 1 above)
Comment (TimBartholomew) [reply tutorials/introduction.ipynb]: Interesting, I remember fixing this. Let me check that all my changes were committed. Yeah I didn't push my changes. Thank you for catching this.

**Review by bknueven — APPROVED** [review]

**Review by srikanthallu — APPROVED** [review]

**Pull request merged**